code/game/gamemodes/cult/talisman.dm

```diff
@@ -151,26 +160,64 @@ Rite of Disorientation
 	if(iscultist(user) && keyword)
 		user << "<b>Keyword:</b> [keyword]"
 
+//Rite of Knowledge: Same as rune, but has two uses
+/obj/item/weapon/paper/talisman/summon_tome
+	cultist_name = "Talisman of Tome Summoning"
+	cultist_desc = "A one-use talisman that will call an untranslated tome from the archives of the Geometer."
+	invocation = "N'ath reth sh'yro eth d'raggathnor!"
+	health_cost = 1
+
+/obj/item/weapon/paper/talisman/summon_tome/invoke(mob/living/user)
+	user.visible_message("<span class='warning'>[user]'s hand glows red for a moment.</span>", \
+						 "<span class='cultitalic'>You speak the words of the talisman!</span>")
+	var/obj/item/weapon/tome/T = new(get_turf(user))
+	if(user.put_in_hands(T))
+		user.visible_message("<span class='warning'>A tome appears in [user]'s hand!</span>", \
+							 "<span class='cultitalic'>An arcane tome materializes in your free hand.</span>")
+	else
+		user.visible_message("<span class='warning'>A tome appears at [user]'s feet!</span>", \
+							 "<span class='cultitalic'>An arcane tome materialzies at your feet.</span>")
 
 //Talisman of Obscurity: Same as rune
 /obj/item/weapon/paper/talisman/hide_runes
 	cultist_name = "Talisman of Veiling"
-	cultist_desc = "A talisman that will make all runes within a small radius invisible, or make invisible runes visible again."
+	cultist_desc = "A talisman that will make all runes within a small radius invisible."
 	invocation = "Kla'atu barada nikt'o!"
 	health_cost = 1
 
 /obj/item/weapon/paper/talisman/hide_runes/invoke(mob/living/user)
 	user.visible_message("<span class='warning'>Dust flows from [user]'s hand.</span>", \
 						 "<span class='cultitalic'>You speak the words of the talisman, veiling nearby runes.</span>")
-	for(var/obj/effect/rune/R in orange(3,src))
-		if(R.invisibility == INVISIBILITY_OBSERVER)
-			R.invisibility = 0
-			R.alpha = initial(R.alpha)
-		else
-			R.visible_message("<span class='danger'>[R] fades away.</span>")
-			R.invisibility = INVISIBILITY_OBSERVER
-			R.alpha = 100 //To help ghosts distinguish hidden runes
+	for(var/obj/effect/rune/R in orange(3,user))
+		R.visible_message("<span class='danger'>[R] fades away.</span>")
+		R.invisibility = INVISIBILITY_OBSERVER
```

Comment:
See above, didn't want any chance of newcult code being incompatable with the oldcult stuff.